css/style.css

```diff
@@ -542,12 +593,31 @@ button:hover {
   align-self: center;
 }
 
+@media (max-width: 56.25em) {
+  .cases__content > h2 {
+    font-size: 2rem;
+    margin: 0;
+  }
```

Comment:
Give this headline some breathing space 🌬

Reply:
Yes, surely!